src/preferences/dialog/dlgprefsound.cpp

```diff
@@ -193,23 +209,53 @@ void DlgPrefSound::slotApply() {
         return;
     }
 
+    m_config.clearInputs();
+    m_config.clearOutputs();
+    emit(writePaths(&m_config));
+
+    EngineMaster::MicMonitorMode configuredMicMonitorMode =
+        static_cast<EngineMaster::MicMonitorMode>(
+              static_cast<int>(m_pMicMonitorMode->get()));
+    if (configuredMicMonitorMode == EngineMaster::MicMonitorMode::DIRECT_MONITOR
+        && m_config.hasMicInputs()) {
+        if (m_pLatencyCompensation->get() == 0.0) {
+            // TODO(Be): Make the "User Manual" text link to the manual in this dialog.
+            QMessageBox measureRoundTripLatencyAdvisoryBox;
+            measureRoundTripLatencyAdvisoryBox.setIcon(QMessageBox::Information);
+            measureRoundTripLatencyAdvisoryBox.setText(tr("Measure Round Trip Latency"));
+            measureRoundTripLatencyAdvisoryBox.setInformativeText(tr("Your microphone input will be out of time in your recorded and broadcasted mixes compared to what you hear. To align them, measure the round trip latency through your setup and enter this value for the Microphone Latency Compensation. Refer to the Mixxx Manual for details."));
```

Comment:
I do not think it makes sense that there are both OK and Apply buttons in the Preferences window. But I don't think we need to deal with that in this PR.